### ISSUE TITLE
help please!

### ISSUE THREAD

**Issue by KENPOKASMMA1982** [ahmedgalal98/Shazam-like-App#1]
hello my friend!congratulations but please can you help me run your amazing project to search music from shazam?

**Comment by ahmedgalal98** [comment]
Sure, please mention the problem you faced. I will  prepare Readme file
explaining how the app work ASAP.

**Comment by KENPOKASMMA1982** [comment]
thanks so much where anser me!tell me please this amazing project is for search music also?

**Comment by KENPOKASMMA1982** [comment]
can you enter with anydesk on my pc to help me how run it?if you want ofcourse!

**Comment by ahmedgalal98** [comment]
No, it is just to find the similarity between the song and database.

**Comment by KENPOKASMMA1982** [comment]
aaa ok thanks mate!

**Comment by ahmedgalal98** [comment]
You are welcome.

**Comment by ahmedgalal98** [comment]
Sure, please mention the problem you faced. I will prepare a Readme file
explaining how the app work ASAP.
